config/sitemap.rb

```diff
@@ -0,0 +1,44 @@
+# Set the host name for URL creation
```

Comment:
Please remove all comments from this file.

Reply:
Done!